proposals/0025/README.md

```diff
@@ -0,0 +1,199 @@
+---
+id: 0025
+title: Tinkerbell on Kubernetes
```

Comment:
Can you rename this something specific, such as `Support for the Kubernetes Resource Model' or 'Support for provisioning via Kubernetes API's"?

Tinkerbell can already run on Kubernetes, which makes this proposal confusing. What you seem to be proposing is buying fully into its resource model:  https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture/resource-management.md - and allowing machines to be provisioned using the Kubernetes API.

Reply:
Thanks! Good callout, I renamed the title